public/views/viewBuildError.html

```diff
@@ -48,7 +48,8 @@
         </tr>
         <tr>
           <td align="left">
-            Found <b>{{cdash.errors.length}}</b>&#x20;{{cdash.errortypename}}s
+            Found <b>{{cdash.numErrors}}</b>&#x20;{{cdash.errortypename}}s
```

Comment:
While we're editing this stuff, let's replace this ``&#x20;`` with a plain old space.